### ISSUE TITLE
Custom module updating frequently breaks all tooltips

### ISSUE THREAD

**Issue by JordanL2** [Alexays/Waybar#890]
Using latest Waybar / Sway on Arch.

With GTK Layer Shell enabled.

I added a custom module that runs an endlessly looping script. I've noticed that if the script outputs an update frequently, say every 0.5s, all Waybar tooltips stop working and won't appear when I hover the mouse over anything. Changing the update frequency to every 0.6s fixes it. This happens even if the script doesn't output a tooltip and the module is customed with "tooltip": "false".

Example config:
```
    "custom/cpu": {
        "exec": "/home/jordan/.config/waybar/cpu_graphical 0.5",
        "format": " {} ",
        "tooltip": true,
        "return-type": "json"
    },
```

Script: https://github.com/JordanL2/LinuxSetup/blob/master/Sway/config/waybar/cpu_graphical

**Comment by JordanL2** [comment]
I'm guessing this is because tooltips have a delay before they appear, and the module updating resets the timer.

**Comment by JordanL2** [comment]
I submitted a patch that partially fixes this issue.

**Comment by Alexays** [comment]
Closed by #905
